libraries/libfc/include/fc/crypto/common.hpp

```diff
@@ -1,18 +1,19 @@
 #pragma once
 #include <fc/crypto/ripemd160.hpp>
-
 #include <fc/reflect/reflect.hpp>
 #include <fc/crypto/base58.hpp>
 #include <fc/crypto/crypto_utils.hpp>
 #include <fc/io/raw.hpp>
 #include <fc/utility.hpp>
 #include <fc/static_variant.hpp>
 
-namespace fc { namespace crypto {
-
-
-
+#include <concepts>
+#include <iterator>
+#include <ranges>
+#include <type_traits>
+#include <cstring>
```

Comment:
nit: move `std` include group above `fc`

Reply:
I disagree. I think `std` should be last.